packages/core/tools/_mixins.scss

```diff
@@ -38,6 +38,20 @@
 //
 
 @mixin visually-hidden() {
+  position: absolute;
```

Comment:
A minor pendantic point but to follow our usual convention, you might want to do is move this to underneath the padding declaration so it's all in alphabetical order and then move the comment outside the mixin.